sdk/nodejs/dagger/api/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { QueryTree } from "./api";
```

Comment:
Same question about this file vs `api/utils.ts`

Reply:
same, old files that somehow stayed.

Reply:
https://github.com/dagger/dagger/pull/3606#discussion_r1022225551